pep-0001.txt

```diff
@@ -188,9 +188,9 @@ The standard PEP workflow is:
   "Informational", or "Process" as appropriate, and for the "Status:"
   field enter "Draft".  For full details, see `PEP Header Preamble`_.
 
-* Update `.github/CODEOWNERS` [7]_ such that any core developer co-author(s) or
-  sponsor are listed for your new file such that any future pull requests will
-  be assigned to them.
+* Update `.github/CODEOWNERS` [7]_ such that any co-author(s) or sponsors
+  with the commit bit on the `PEP repository`_ are listed for your new file
+  such that any future pull requests will be assigned to them.
```

Comment:
```suggestion
  with write access to the `PEP repository`_ are listed for your new file,
  such that any future pull requests will be assigned to them.
```

Add missing comma (from original). Also, I suggest using the actual term here as used in the GitHub UI, as its meaning is clearer and more obvious than old-school git jargon that not every user may be familiar with.

Reply:
While we're here, maybe we should reword the sentence such that it does not say "such that" twice.

Reply:
Also applied to useages of "git push privileges" for consistency